common/templates/course_modes/choose.html

```diff
@@ -71,6 +71,33 @@ <h3 class="title">${_("Sorry, there was an error when trying to enroll you")}</h
                             <div class="register-choice register-choice-certificate">
                                 <div class="wrapper-copy">
                                     <span class="deco-ribbon"></span>
+                                    % if has_credit_upsell:
+                                    <h4 class="title">${_("Pursue Academic Credit with a Verified Certificate")}</h4>
+
+                                    <div class="copy">
+                                        <p>${_("Become eligible for academic credit and highlight your new skills and knowledge with a verified certificate. Use this valuable credential to qualify for academic credit from {org}, advance your career, or highlight in school applications.").format(org=course_org)}</p>
```

Comment:
Minor: The second sentence in this copy has a grammatical error. "Use this valuable credential to qualify for academic credit from {org}, advance your career, or highlight in school applications" should instead be something like "Use this valuable credential to qualify for academic credit from {org}, advance your career, or highlight **it** in school applications." Has this copy passed a docs review?